File: ctime.c
#include<stdio.h>
#include<stdlib.h>
#include <time.h>
int main () {

    clock_t startTime;
    startTime = clock();
    int i;
    for (i = 0; i <= 2000000000; i++) {
        
    }
    clock_t endTime;
    endTime = clock();

    float totalTime = endTime - startTime;
	printf("%f\n", totalTime/1000);

    return 0;

}

File: ctime.java
class ctime {
 public static void main (String[] args) {

    long startTime = System.currentTimeMillis();
    int i;
    
    for (i = 0; i < 2000000000; i++) {
    }
    long endTime = System.currentTimeMillis();
    float totalTime = (endTime - startTime);
    System.out.println("time: " + totalTime/1000);
    
 }


}
